model.py

```diff
@@ -9032,8 +9032,12 @@ class ExternalIntegration(Base):
     AXIS_360 = DataSource.AXIS_360
     ONE_CLICK = DataSource.ONECLICK
 
+    # These protocols are only used on the Content Server.
```

Comment:
Spell out that this is importing texts from a Project Gutenberg mirror, not importing Gutenberg texts from the content server via OPDS.